arangod/Agency/State.cpp

```diff
@@ -788,8 +790,23 @@ bool State::checkCollection(std::string const& name) {
   return true;
 }
 
+/// Drop
+void State::dropCollection(std::string const& colName) {
+  try {
+    auto col = _vocbase->lookupCollection(colName)->id();
+    auto res = _vocbase->dropCollection(col, false, -1.0).errorNumber();
+    if (res != TRI_ERROR_NO_ERROR) {
+      LOG_TOPIC("ba841", ERR, Logger::AGENCY)
+        << "unable to drop collection log: " << TRI_errno_string(res);
+    }
+  } catch (std::exception const& e) {
+    LOG_TOPIC("69fc4", FATAL, Logger::REPLICATION)
+      << "unable to drop collections " << colName << ": " << e.what();
```

Comment:
```suggestion
      << "unable to drop collection '" << colName << "': " << e.what();
```

Reply:
It is a bit confusing that in both error case (res.fail() or exception) there will be a FATAL log message but the program will continue happily. Do we want to abort here?
If not, then we need to change the log level from FATAL to ERR.

Reply:
Done, we exit here. Log message also corrected.